docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -237,3 +237,27 @@ If you enable small scanner on large size scan incorrectly, then you are likely
 | *Deprecated In Release*     | Not applicable.
 |===
 
+<<<
+[[scratch-disks]]
+== SCRATCH_DISKS
```

Comment:
There is no SCRATCH_DISKS CQD in Trafodion. There is a SCRATCH_DIRS CQD, which gives a colon-separated list of directories to use for scratch space. I'm guessing that's what was intended here.